taskcluster/fxci_config_taskgraph/transforms/integration_test.py

```diff
@@ -198,6 +252,19 @@ def schedule_tasks_at_index(config, tasks):
         return
 
     for task in tasks:
+        include_attrs = task.pop("include-attrs", {})
+        exclude_attrs = task.pop("exclude-attrs", {})
+        include_deps = task.pop("include-deps", [])
```

Comment:
This is getting complex enough that's it's probably worth adding a schema.

Reply:
Good call....